src/main/resources/assets/wurst/lang/ru_ru.json

```diff
@@ -117,23 +118,32 @@
+  "description.wurst.hack.tired": "Делает Вас как выглядел Alexander в Апреле 2015.\nВидно только другим игрокам.",
+  "description.wurst.hack.toomanyhax": "Отключает ненужные вам функции.\nПозволяет быть уверенным что вы не включите случайно что-то не то и получите бан.\nДля тех, кто хочеть почитерить \"совсем чуть-чуть\".\n\nИспользуйте команду §6.toomanyhax§r чтобы выбрать какие функции следует отключить.\nНапишите §6.help toomanyhax§r для более подробной информации.",
   "description.wurst.hack.tp-aura": "Автоматически атакует ближайщую сущность при этом телепортируясь вокруг неё.",
-  "description.wurst.hack.trajectories": "Предсказывает траекторию полета стрел или бросаемых вещей.",
-  "description.wurst.hack.treebot": "Экспериментальная функция, бот, который ходит и автоматически вырубает деревья.\nПока что ограничен обычными маленькими деревьями.",
-  "description.wurst.hack.triggerbot": "Автоматически атакует сущность на которую вы посмотрите.",
-  "description.wurst.hack.trollpotion": "Создает зелье с множеством противных эффектов. Для работы нужен креатив.",
+  "description.wurst.hack.trajectories": "Показывает траекторию полета стрел и бросаемых вещей.",
+  "description.wurst.hack.treebot": "Экспериментальная функция, бот который ходит и автоматически вырубает деревья.\nПока что ограничен обычными маленькими деревьями.",
+  "description.wurst.hack.triggerbot": "Автоматически атакует сущность на которую Вы смотрите.",
+  "description.wurst.hack.trollpotion": "Создает зелье с множеством противных эффектов. Для работы требуется креатив.",
   "description.wurst.hack.truesight": "Позволяет видеть невидимые сущности.",
-  "description.wurst.hack.tunneller": "Автоматически копает тунель.\n\n§c§lВНИМАНИЕ:§r Хоть бот и будет пытаться избегать попадания в лаву и других опасностей, нет гарантий что он не умрет. Посылайте его копать только с вещами которые вам было бы не жалко потерять.",
-  "description.wurst.hack.x-ray": "Позволяет видеть руды и другие полезности через стены."
+  "description.wurst.hack.tunneller": "Автоматически копает туннель.\n\n§c§lВНИМАНИЕ:§r Хоть бот и будет пытаться избегать лаву и другие опасности, нет гарантии что он не умрет. Посылайте его копать только с вещами которые Вам было бы не жалко потерять.",
+  "description.wurst.hack.x-ray": "Позволяет видеть руды и другие блоки сквозь стены.",
+  "description.wurst.altmanager.premium": "Этот аккаунт имеет пароль и может присоединяться к любым серверам.",
+  "description.wurst.altmanager.cracked": "Этот аккаунт не имеет пароля и он сможет присоединяться только к серверам без авторизации.",
+  "description.wurst.altmanager.failed": "Последняя попытка зайти в этот аккаунт была неудачной.",
+  "description.wurst.altmanager.checked": "Пароль работал в предыдущий раз.",
```

Comment:
Does this say "The password worked last time."? It is supposed to say that the password has worked _at some point_ in the past, but not necessarily _the last time_.

It is possible for an alt to have both `checked` and `failed` flags at the same time, in which case both of these strings are displayed together to say something like "This alt used to work in the past, but the most recent login attempt has failed".

Reply:
Fixed, now it says that password worked at some point, not the last time.